src/Microsoft.VisualStudio.Editor.Razor/BackgroundParser.cs

```diff
@@ -345,10 +345,35 @@ private void WorkerLoop()
 
             private RazorCodeDocument ParseChange(ITextSnapshot snapshot, CancellationToken token)
             {
+                const int MaxRetryCount = 5;
+
                 EnsureOnThread();
 
                 var sourceDocument = new TextSnapshotSourceDocument(snapshot, _filePath);
-                var imports = _templateEngine.GetImports(_filePath);
+
+                IEnumerable<RazorSourceDocument> imports = null;
+                var retries = 0;
+                while (retries++ < MaxRetryCount)
```

Comment:
This is a tactical fix since it's for 15.6 RTM. In 15.7 + we'll do a more complete fix that logs errors if it can't access an import etc.